apps/web/src/components/sidebar/NestedSafeInfo/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Tooltip, SvgIcon, Typography, List, ListItem, Box, ListItemAvatar, Avatar, ListItemText } from '@mui/material'
+import CheckIcon from '@mui/icons-material/Check'
+import type { ReactElement } from 'react'
+
+import NestedSafesIcon from '@/public/images/sidebar/nested-safes-icon.svg'
+import NestedSafes from '@/public/images/sidebar/nested-safes.svg'
+import InfoIcon from '@/public/images/notifications/info.svg'
+
+export function NestedSafeInfo(): ReactElement {
+  return (
+    <Box display="flex" flexDirection="column" alignItems="center" pt={1}>
+      <NestedSafes />
+      <Box display="flex" gap={1} py={2}>
+        <Typography fontWeight={700}>No Nested Safes yet</Typography>
+        <Tooltip
+          title="Nested Safes are separate wallets owned by your main Account, perfect for organizing different funds and projects."
+          placement="top"
+          arrow
+          sx={{ ml: 1 }}
+        >
+          <span>
+            <SvgIcon
+              component={InfoIcon}
+              inheritViewBox
+              fontSize="small"
+              color="border"
+              sx={{ verticalAlign: 'middle' }}
+            />
+          </span>
+        </Tooltip>
+      </Box>
+      <Box display="flex" gap={2} alignItems="center" pt={1} pb={4}>
+        <Avatar sx={{ padding: '20px', backgroundColor: 'success.background' }}>
+          <SvgIcon component={NestedSafesIcon} inheritViewBox color="primary" sx={{ fontSize: 20 }} />
+        </Avatar>
+        <Typography variant="body2" fontWeight={700}>
+          Nested Safes allow you to:
+        </Typography>
+      </Box>
+      <List sx={{ p: 0, display: 'flex', flexDirection: 'column', gap: 2 }}>
+        {[
+          'rebuild your organizational structure onchain',
+          'explore new DeFi opportunities without exposing your main Account',
+          'deploy specialized modules and enable easier access through passkeys and other signers',
```

Comment:
Replace the 44  with "deploy specialized modules and extend Safe functionality"

Reply:
Changed in e5021b56804135a71b37c739f9f09c50141a9ad8.